### ISSUE TITLE
Support for comparing 3 benchmarks (up from 2)

### ISSUE THREAD

**Issue by robinheghan** [BrianHicks/elm-benchmark#21]
Today one can compare 2 benchmarks against each other. This is great when one want's to compare old code (TM) to new code (TM) and see if you're actually making things better or not.

Once in a while though, it makes sense to compare up to three different things at the same time, which elm-benchmark does not support.

Let's say I'm working on a new data-structure. It makes sense to compare to a similar data-structure, to justify why a new data-structure is needed, but it might also make sense to compare to another implementation. OR it makes sense to compare against another data-structure, as well as old code (TM) and new code (TM). Or perhaps you've found three implementations for a thing, and you want to check which one is better for your use case.

tl; dr;
Allowing the comparison of three benchmarks, allows one to compare new code against previous code, as well as against a base line.

I can also see the use case for comparing 4 benchmarks as well, but that is a very narrow use case of comparing three data-structures for one use case. I think todays comparison of two benchmarks supports 85% of all use cases, 3 comparisons 14%, and 4 would be a whopping 1%. At least for me.
